### PR TITLE
Argocd support

### PR DESCRIPTION
**Pull request by chris-bridgett-nandos**

### CODE REVIEW

**Comment by dinagraves** [comment]
Hi Chris!  The python tests failed due to this bug in the github action we were using. https://github.com/excitedleigh/setup-nox/issues/419. -- Tests have been updated if you want to re-trigger!

**Comment by chris-bridgett-nandos** [comment]
> Hi Chris! The python tests failed due to this bug in the github action we were using. [excitedleigh/setup-nox#419](https://github.com/excitedleigh/setup-nox/issues/419). -- Tests have been updated if you want to re-trigger!

Hi @dinagraves, sorry, I don't think I can:

> First-time contributors need a maintainer to approve running workflows.

Would you mind re-running it? 🙂

File: bq-workers/argocd-parser/main.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+@app.route("/", methods=["POST"])
+def index():
+    """
+    Receives messages from a push subscription from Pub/Sub.
+    Parses the message, and inserts it into BigQuery.
+    """
+    event = None
+    envelope = request.get_json()
```
Comment (dinagraves) [inline bq-workers/argocd-parser/main.py:33]:
```suggestion
    if not request.is_json:
        raise Exception("Expecting JSON payload")
    envelope = request.get_json()
```

Comment (dinagraves) [reply bq-workers/argocd-parser/main.py:33]: Updated the way the envelope is handled! See example here -- https://github.com/GoogleCloudPlatform/fourkeys/blob/main/bq-workers/circleci-parser/main.py#L34

Comment (chris-bridgett-nandos) [reply bq-workers/argocd-parser/main.py:35]: Great, thanks. 🙂

**Comment by martasd** [comment]
Hi @dinagraves. What is the status of this PR? Is there anything still needed before it can be merged?

**Comment by dinagraves** [comment]
Apologies for the delay!  It's merged!  Thank you!

**Pull request merged**